contracts/leaser/src/migrate.rs

```diff
@@ -77,7 +108,7 @@ mod test {
         ];
         assert_eq!(
             Err(ContractError::ParseError { err: err.into() }),
-            super::migrate_leases(no_leases.into_iter(), new_code)
+            super::migrate_leases(no_leases.into_iter(), 5, new_code)
         );
```

Comment:
Expected expressions are second arguments to the macro.
```suggestion
        assert_eq!(
            super::migrate_leases(no_leases.into_iter(), 5, new_code),
            Err(ContractError::ParseError { err: err.into() })
        );
```

Reply:
same